### PR TITLE
Doc/264 update tco

### PR DESCRIPTION
**Pull request by Marchabar**
# Description
Added the amazon aws to the deployment costs.
# Checklist:
- [X] I have performed a self-review of my code running `npm run build` and `npm run serve` after pushing the code.
- [X] I have checked that there is no fragile information published unintentionally.

# Screenshot:
image
image

### CODE REVIEW

File: docs/03. Planificación/3.8. TCO.mdx [hunk 1]
```diff
@@ -182,7 +192,7 @@ En caso de que cambiasen las políticas de uso de la API de SendGrid o se supera
 ### 3.3 Costes de GDPR
 En este sistema, se tratan datos sensibles y personales sobre familiares o personas. Esto hace necesario la introducción de una serie de roles y responsabilidades para garantizar el cumplimiento de las normativas de protección de datos, Reglamento General de Protección de Datos (GDPR por sus siglas en inglés). 
 
-En nuestro caso, donde el sistema solamente va a ser utilizado por los administradores, y donde no va a haber una gran cantidad de usuarios, suponemos que es suficiente con la introducción de un Data Protection Officer (DPO). Suponemos también un sueldo medio de 22.50€ por hora, y una dedicación semanal de 10 horas. Por lo tanto, el coste mensual de GDPR es de: 22.50 x 10 x 4 = 900.00€
+En nuestro caso, donde el sistema solamente va a ser utilizado por los administradores, y donde no va a haber una gran cantidad de usuarios, suponemos que es suficiente con la introducción de un GPRD Officer. Suponemos también un sueldo medio de 22.50€ por hora, y una dedicación semanal de 10 horas. Por lo tanto, el coste mensual de GDPR es de: 22.50 x 10 x 4 = 900.00€
```
Comment (alvgonfri) [inline docs/03. Planificación/3.8. TCO.mdx:195]: I think you mean "GDPR Officer".

**Review by alvgonfri — CHANGES_REQUESTED** [review]
It looks right, I just requeted a small change.

File: docs/03. Planificación/3.8. TCO.mdx (hunk 1 above)
Comment (Marchabar) [reply docs/03. Planificación/3.8. TCO.mdx]: Fixed!

**Review by alvgonfri — APPROVED** [review]
Perfect!

**Pull request merged**